src/shared/lib/generateMetadataObject.ts

```
@@ -1,24 +1,28 @@
   images?: string[];
 };
 
 export const generateMetadataObject = ({
-  title = DEFAULT_TITLE,
-  description = DEFAULT_DESCRIPTION,
+  titleKey = DEFAULT_TITLE_KEY,
+  descriptionKey = DEFAULT_DESCRIPTION_KEY,
   url = DEFAULT_URL,
   images = [DEFAULT_IMAGE],
 }: Args = {}): Metadata => {
+  const { t } = useTranslation("common");
```

ESLint

[error] React Hook "useTranslation" is called in function "generateMetadataObject" that is neither a React function component nor a custom React Hook function. React component names must start with an uppercase letter. React Hook names must start with the word "use".

(react-hooks/rules-of-hooks)